tests/test_cli.py

```diff
@@ -495,6 +495,46 @@ def test_types(cli: click.testing.CliRunner) -> None:
         assert "✓ success2" in result.stdout
 
 
+def test_env(cli: click.testing.CliRunner) -> None:
+    with cli.isolated_filesystem():
+        with open("riotfile.py", "w") as f:
+            f.write(
+                """
+from riot import Venv
+
+venv = Venv(
+    pkgs={
+        "pytest": [""],
+    },
+    venvs=[
+        Venv(
+            env={"foobar": "baz"},
+            pys=[3],
+            name="envtest",
+            command="pytest",
+        ),
+    ],
+)
+            """
+            )
+
+        with open("test_success.py", "w") as f:
+            f.write(
+                """
+import os
+
+def test_success():
+    assert os.environ["foobar"] == "baz"
+            """
+            )
+
+        result = cli.invoke(
+            riot.cli.main, ["run", "-s", "envtest"], catch_exceptions=False
```

Comment:
This looks great! I had forgotten about #105. It would be good here to also have a similar test to what I had added for to confirm the `os.environ` doesn't get passed along with the instance env you are testing here by default and only does when `--pass-env` is provided.

Reply:
#122